### PR TITLE
:hammer: improvement of APIs

### PR DESCRIPTION
**Pull request by tonnyhideyori**
- [x] improve createPorject method 
- [x] added new methods and edit some existing codes and bug fix

### CODE REVIEW

File: apicomponents/bom.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import base64,json
```
Comment (manuel-sommer) [inline apicomponents/bom.py:1]: do you really need to import it here? It is imported already in main.py

Comment (tonnyhideyori) [reply apicomponents/bom.py:1]: No we have to import in the file it is used hence I will remove it from the main.py

File: main.py [hunk 2]
```diff
@@ -107,3 +107,7 @@ def get_analysis(self, project, component, vulnerability):
             return response.status_code
 
 
+dt=DependencyTrackAPI("http://localhost:8081","Z0g4jvxF1Yek3R8balySFj5kGahjR3oj")
```
Comment (manuel-sommer) [inline main.py]: @tonnyhideyori  Please remove those lines. Thank you for your contribution

**Review by manuel-sommer — COMMENTED** [review]
thank you @tonnyhideyori

**Pull request merged**